packages/webapp/pages/onboarding.tsx

```diff
@@ -438,109 +438,108 @@ export function OnboardPage(): ReactElement {
   }
 
   return (
-    <div
-      className={classNames(
-        'z-3 flex h-full max-h-dvh min-h-dvh w-full flex-1 flex-col items-center overflow-x-hidden',
-        layout.hasCta && 'fixed',
-      )}
-    >
-      {showOnboardingPage && (
-        <img
-          alt="Onboarding background"
-          className="pointer-events-none absolute inset-0 -z-1 h-full w-full object-cover tablet:object-center"
-          fetchPriority="high"
-          loading="eager"
-          role="presentation"
-          src={onboardingVisual.fullBackground.mobile}
-          srcSet={`${onboardingVisual.fullBackground.mobile} 450w, ${onboardingVisual.fullBackground.desktop} 1024w`}
-          sizes="(max-width: 655px) 450px, 1024px"
-        />
-      )}
-      <Toast autoDismissNotifications={autoDismissNotifications} />
-      {showGenerigLoader && <GenericLoader />}
-      <OnboardingHeader
-        showOnboardingPage={showOnboardingPage}
-        setAuth={setAuth}
-        customActionName={customActionName}
-        onClick={onClickCreateFeed}
-        activeScreen={activeScreen}
-        showPlusIcon={isPlusCheckout && activeScreen === OnboardingStep.Plus}
-      />
+    <PaymentContextProvider>
       <div
         className={classNames(
-          'flex w-full flex-grow flex-col flex-wrap justify-center px-4 tablet:flex-row tablet:gap-10 tablet:px-6',
-          activeScreen === OnboardingStep.Intro && wrapperMaxWidth,
-          !isAuthenticating && 'mt-7.5 flex-1 content-center',
-          [OnboardingStep.Extension, OnboardingStep.InstallDesktop].includes(
-            activeScreen,
-          ) && '!flex-col',
+          'z-3 flex h-full max-h-dvh min-h-dvh w-full flex-1 flex-col items-center overflow-x-hidden',
+          layout.hasCta && 'fixed',
         )}
       >
         {showOnboardingPage && (
-          <>
-            <div className="mt-5 flex flex-1 flex-grow-0 flex-col tablet:mt-0 tablet:flex-grow laptop:mr-8 laptop:max-w-[27.5rem]">
-              <OnboardingHeadline
-                className={{
-                  title: 'tablet:typo-mega-1 typo-large-title',
-                  description: 'mb-8 typo-body tablet:typo-title2',
-                }}
-              />
-              <AuthOptions {...authOptionProps} />
-            </div>
-            <SignupDisclaimer className="mb-0 tablet:mb-10 tablet:hidden" />
-          </>
+          <img
```

Comment:
What actually changed here? looks like a lot, but probably due to wrapping?
Just note this def needs critical flow testing on deployment.

Reply:
Yes, just wrapping: https://github.com/dailydotdev/apps/pull/4209#discussion_r1963286424